### DISCUSSION TITLE
Middleware file does not exist

### DISCUSSION THREAD

**Discussion by sybanyt** [htpcBeginner/docker-traefik#268] [Support]
I am unable to start up the container. I am following the traefik2 guide and starting with only traefik. In the docker logs I am getting the error:
2022-04-24T23:14:32.900304553Z time="2022-04-24T23:14:32Z" level=info msg="Configuration loaded from flags."
2022-04-24T23:14:35.508591200Z time="2022-04-24T23:14:35Z" level=error msg="middleware \"chain-oauth@file\" does not exist" entryPointName=https routerName=traefik-rtr@docker

I have created a middelwares.toml file in the traefik2/rules folder with the content below:
[http.middlewares]
  [http.middlewares.chain-oauth]
    [http.middlewares.chain-oauth.basicAuth]
#      users = [
#        "user:$apsdfswWvC/6.$E3FtsfTntPC0wVJ7IUVtX1",
#      ]
      realm = "Traefik2 Basic Auth"
      usersFile = "/shared/.htpasswd" #be sure to mount the volume through docker-compose.yml

**Comment by sybanyt** [comment]
Figured it out. I was creating the middlewares.toml file in the wrong directory

**Reply by CaoHoa1** [reply]
I got the same error as you, time="2022-12-02T09:11:25Z" level=info msg="Configuration loaded from flags." can you show me where should i put that file?
